src/WpThemeMagicPropertiesClassReflectionExtension.php

```diff
@@ -31,6 +31,6 @@ public function hasProperty(ClassReflection $classReflection, string $propertyNa
     // phpcs:ignore SlevomatCodingStandard.Functions.UnusedParameter
     public function getProperty(ClassReflection $classReflection, string $propertyName): PropertyReflection
     {
-        return new DummyPropertyReflection();
+        return new DummyPropertyReflection(); // @phpstan-ignore-line ignore that this is not covered by BC
```

Comment:
```suggestion
        // @phpstan-ignore-next-line This is not covered by BC.
        return new DummyPropertyReflection();
```

Reply:
What is the real solution instead of `DummyPropertyReflection`?

Reply:
I don't know unfortunately, my knowledge of phpstan internals/extensions is unfortunately very limited too. Maybe it can be adapted in a follow-up

I force-pushed and adapted the ignore